azuredeploy.json

```diff
@@ -7,13 +7,24 @@
             "defaultValue": "[concat('HealthBot', uniqueString(resourceGroup().name, utcNow('F')))]",
             "metadata":{
                 "description": "Web site name. Has to be unique."
-                
+
+            }
+        },
+        "serverKind": {
+            "type": "string",
+            "defaultValue": "linux",
+            "allowedValues": [
+                "linux",
+                "windows"
+            ],
+            "metadata": {
+                "description": "Host type: Linux or Windows"
```

Comment:
Maybe we can also state here in the description that linux is recommended? 
something like "Host type: Linux or Windows (Linux is recommended)"
@GuyBeckerMicrosoft  @tamirkamara

Reply:
I've added it in the readme, I think its more natural to have it there, compared to a long termplate users rarely read

Reply:
fixed

Reply:
BTW, I don't think this is displayed anywhere. Have you seen something?

Reply:
no. but it might be client related. maybe with CLI it does appear...